spec/dry/types/schema_spec.rb

```diff
@@ -501,4 +501,45 @@ def self.meta
       specify { expect(type.transform_keys?).to be(true) }
     end
   end
+
+  describe '#merge' do
+    it 'adds keys and values from given schema' do
+      schema_1 = Dry::Types['hash'].schema(name: 'string')
+      schema_2 = Dry::Types['hash'].schema(age: 'integer')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use normalcase for variable numbers.](https://app.codacy.com/gh/dry-rb/dry-types/pullRequest?prid=4765184)